### DISCUSSION TITLE
金融知识小助手

### DISCUSSION THREAD

**Discussion by wuhaostudio** [InternLM/Tutorial#279] [project]
借鉴复旦大学开源的DISC-FinLLM模型，对InternLM-7B模型进行微调，结合Lagen框架，构建可以自动web搜索的基础智能助手，功能如下：
金融咨询：该模组可以在中国金融语境下，与用户展开关于金融话题的多轮对话，或是为用户解释金融专业的相关知识。

金融文本分析：该模组可以帮助用户在金融文本上完成信息抽取、情感分析、文本分类、文本生成等NLP任务。

金融计算：该模组可以帮助用户完成与数学计算相关的任务，除了利率、增长率等基本计算。

金融知识检索问答：该模组可以基于金融新闻、研报和相关政策文件为用户提供投资建议、时事分析、政策解读

小组成员：ARCHAO1225
                  Rogue-Trader-zzy
                  Practicing7
                  conno

**Comment by abelezm** [comment]
在吗，我可以报名吗

**Reply by wuhaostudio** [reply]
抱歉，已满员

**Comment by SiliconSisyphus** [comment]
你好，请问队伍满员了吗

**Reply by wuhaostudio** [reply]
抱歉，满员了

**Comment by JinProton** [comment]
你好，请问还可以加入吗

**Reply by wuhaostudio** [reply]
抱歉，满员了

**Comment by raulming** [comment]
可以加入吗？

**Reply by wuhaostudio** [reply]
抱歉，满员了

**Comment by LiyanJin** [comment]
可以加入吗？

**Reply by wuhaostudio** [reply]
抱歉，满员了
